sda/internal/database/db_functions.go

```diff
@@ -202,34 +202,37 @@ func (dbs *SDAdb) storeHeader(header []byte, id string) error {
 }
 
 // SetArchived marks the file as 'ARCHIVED'
-func (dbs *SDAdb) SetArchived(file FileInfo, fileID, corrID string) error {
-	var (
-		err   error
-		count int
-	)
+func (dbs *SDAdb) SetArchived(file FileInfo, fileID string) error {
+	var err error
 
-	for count == 0 || (err != nil && count < RetryTimes) {
-		err = dbs.setArchived(file, fileID, corrID)
-		count++
+	for count := 1; count <= RetryTimes; count++ {
+		err = dbs.setArchived(file, fileID)
+		if err == nil {
+			break
+		}
+		time.Sleep(time.Duration(math.Pow(2, float64(count))) * time.Second)
 	}
 
 	return err
 }
-func (dbs *SDAdb) setArchived(file FileInfo, fileID, corrID string) error {
+func (dbs *SDAdb) setArchived(file FileInfo, fileID string) error {
```

Comment:
no longer updates the `file_event_log`?

Reply:
If you look in ingest.go where this is called from you will see that the file_event_log update is done separately.